src/Sylius/Behat/Context/Ui/Admin/ManagingProductReviewsContext.php

```diff
@@ -239,27 +266,39 @@ public function thisProductReviewTitleShouldBeTitled($productReviewTitle)
     /**
      * @Then /^(this product review) should still have a comment "([^"]+)"$/
      */
-    public function thisProductReviewShouldStillHaveAComment(ReviewInterface $productReview, $comment)
+    public function thisProductReviewShouldStillHaveAComment(ReviewInterface $productReview, string $comment): void
     {
         $this->iWantToModifyTheProductReview($productReview);
 
         $this->assertElementValue('comment', $comment);
     }
 
     /**
-     * @param string $element
-     * @param string $value
+     * @Then the first product review in the list should have title :title
      */
-    private function assertElementValue($element, $value)
+    public function theFirstProductReviewInTheListShouldHaveTitle(string $title): void
     {
-        Assert::true($this->updatePage->hasResourceValues([$element => $value]));
+        $titles = $this->indexPage->getColumnFields('title');
+
+        Assert::contains(reset($titles), $title);
```

Comment:
```suggestion
        $titles = $this->indexPage->getColumnFields('title');
        Assert::notFalse($titles, 'There are no reviews.');

        Assert::contains(reset($titles), $title);
```
Faster feedback loop and a bit more context for the user, same case with the `end` call within `theLastProductReviewInTheListShouldHaveTitle`.

Reply:
If there are no rows, the exception will be thrown earlier by `TableAccessor::getIndexedColumn`